docs/sources/alerting/set-up/configure-high-availability/_index.md

```diff
@@ -1,9 +1,11 @@
 ---
 aliases:
   - ../high-availability/enable-alerting-ha/
-  - ../unified-alerting/high-availability/
+  - ../unified-alerting/high-availability/ ## latest/alerting/unified-alerting/high-availability
+  - ../high-availability/ ## latest/alerting/high-availability
+  - ../fundamentals/high-availability/ ## latest/alerting/fundamentals/high-availability
```

Comment:
Ideally these comments should be the full URL path from the site root.

Reply:
fixed by https://github.com/grafana/grafana/pull/84248/commits/1133f2d73d48d096fedd2b7f76eec40923e86344